openmdao/utils/mpi.py

```diff
@@ -57,31 +57,37 @@ def use_proc_files():
             ofile = open(os.path.join(working_dir, "%d.out" % MPI.COMM_WORLD.rank), 'wb')
         _redirect_streams(ofile.fileno())
 
+# Determine whether OPENMDAO_REQUIRE_MPI is set. Use 'try' as the default value. 
+use_mpi = 'try'
+
+if 'OPENMDAO_REQUIRE_MPI' in os.environ:
+    use_mpi = os.environ['OPENMDAO_REQUIRE_MPI']
```

Comment:
Going forward, do we need to put `OPENMDAO_REQUIRE_MPI` in our environments? If so, we should probably add that requirement to the mpi documentation. (however, I don't think we have any general mpi/parallel documentation...)

Reply:
It's not required - the way it's implemented, it will try to use MPI by default, but revert back if it can't import mpi4py or MPI.COMM_WORLD.size == 1. The env var gives the user some control over the behavior